18 Normalizr/src/schemas/countrySchema.ts

```diff
@@ -0,0 +1,4 @@
+import { schema } from 'normalizr';
+
+export const countrySchema = new schema.Entity('countries');
```

Comment:
Just in case it scales we could go for a solution like

```
+ src
    + schema
         * index.ts
         * country.ts
         * student.ts
```

Then just in index we export all the schemas.

If somebody has to use this he only has to worry about

```
import "./schema"
```

But on the name of the schema it self, not sure if we should keep the "Schema" prefix, to avoid creating aliases if there are name collisions.